Interfaces/API/Dirac.py

```diff
@@ -1674,7 +1674,7 @@ def getOutputSandbox(self, jobID, outputDir=None, oversized=True, noJobDir=False
 
     # New download
     result = SandboxStoreClient(useCertificates=self.useCertificates).downloadSandboxForJob(jobID, 'Output', dirPath,
-                                                                                            unpack)
```

Comment:
To avoid these kind of problems, use keyword-argument: `inMemory=False, unpack=unpack`